client/hotrod-client/src/main/java/org/infinispan/client/hotrod/logging/Log.java

```diff
@@ -371,4 +371,8 @@
    @LogMessage(level = WARN)
    @Message(value = "Failed to load and create an optional ProtoStream serialization context initializer: %s", id = 4104)
    void failedToCreatePredefinedSerializationContextInitializer(String className, @Cause Throwable throwable);
+
+   @LogMessage(level = WARN)
+   @Message(value = "All the servers are marked as failed. Cluster might have completely shut down, try reverting to the initial server list.", id = 4105)
```

Comment:
Should this be ", attempting to revert to the initial ...". As this is now a WARN message, I don't want users to have to think they need to take some form of action which is implied by "try reverting".

Reply:
I opted to remove "try" instead, since the client is always reverting to the initial list -- there's no way the action could fail and the client could stick with the current list.